docs/release-notes/8.5.asciidoc

```diff
@@ -1,6 +1,21 @@
 [[release-notes-header-8.5.0]]
 == 8.5
 
+[discrete]
+[[release-notes-8.5.1]]
+=== 8.5.1
+
+[discrete]
+[[bug-fixes-8.5.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that caused {elastic-endpoint}s (version 8.5.0) running on Linux systems with many CPUs to sometimes become unhealthy https://github.com/elastic/endpoint/issues/34[#34].
+* Fixes bug that caused incorrect alerts to display in Timeline when investigating alerts from the Detection & Response dashboard ({pull}144319[#144319]).
+* Updates the User authentication area chart so it can be opened in Lens ({pull}144011[#144011]).
+* Displays the {jira} connector icon for users with a basic license ({pull}143916[#143916]).
```

Comment:
The fix looks like this but in fact it replaces the icon since the previous one didn't handle greying out properly which happens under a basic license. It's fine to leave the message as is.

Reply:
Using the following as discussed in Slack:

```suggestion
* Fixes the {jira} connector icon for users with a basic license ({pull}143916[#143916]).
```